src/averageAges.js

```diff
@@ -37,7 +43,23 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren ? people.filter((el) => {
+    for (const obj of people) {
+      if (el.name === obj.mother) {
+        return true;
+      }
```

Comment:
use array method for this

Reply:
didn't figure out the way to do this using array method

Reply:
mb you could use `find()`